sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/documentanalysis/models/DocumentBarcode.java

```diff
@@ -57,7 +57,7 @@ public String getValue() {
      *
      * @return the polygon value.
      */
-    public List<Float> getBoundingPolygon() {
+    public List<Point> getBoundingPolygon() {
```

Comment:
Changing from `Float` to `Point` is a breaking change.

Reply:
This was introduced in beta so isn't breaking change.